src/wallet/wallet2.cpp

```diff
@@ -5126,8 +5125,18 @@ std::string wallet2::exchange_multisig_keys(const epee::wipeable_string &passwor
       ""
     };
 
+  // KLUDGE: early return if there are no kex messages and main kex is complete (will return the post-kex verification round
+  //         message) (it's a kludge because this behavior would be more appropriate for a standalone wallet method)
+  if (kex_messages.size() == 0)
```

Comment:
Yes, because existing multisig wallets are stuck, unable to complete the post-kex verification round that was added in the v0.18 (an oversight in testing the multisig changes).

Reply:
They aren't guaranteed to be stuck though, right? Seems like this would only happen if other signers lost someone's post-kex message? If that could happen, couldn't any of the steps get stuck? I guess that's why the force option was added?

Reply:
The post-kex round did not exist before v0.18, and the only way to get a post-kex message right now is from `exchange_multisig_keys()` as the output of a kex round. I guess I am overloading this PR a little so we can fix multisig faster.